### PR TITLE
Add get property type method

### PR DESCRIPTION
**Pull request by submarcos**

### CODE REVIEW

File: geostore/models.py [hunk 1]
```diff
@@ -390,6 +391,17 @@ def get_property_title(self, prop):
...
+    def get_property_type(self, prop):
+        """ Get json property type with its name """
+        json_form_properties = self.schema.get('properties', {})
+
+        if prop in json_form_properties:
+            data = json_form_properties[prop]
+            prop_type = data.get('type')
+            return prop_type
+
+        return prop
```
Comment (nahuelange) [inline geostore/models.py]: The behavior returning the prop name instead of a type seems weird, how is it explained ?

Comment (submarcos) [reply geostore/models.py]: yes. bad copy / paste. thx

Comment (nahuelange) [reply geostore/models.py]: Isn't a test for this case missing ?

**Review by nahuelange — APPROVED** [review]

File: geostore/models.py (hunk 1 above)
Comment (submarcos) [reply geostore/models.py]: done

**Pull request merged**